lib/wpcom.js

```diff
@@ -29,7 +30,7 @@ var wpcom = {
 				rejectUnauthorized: false,
 			};
 
-			logger.trace( 'setting blogid ' + o_request.blog_id + ' status ' + o_request.status_id );
+			logger.trace( 'setting blogid ' + o_request.blog_id + ' status ' + o_request.status_id + ', URL: ' + serverObject.monitor_url );
```

Comment:
Minor. 
Feels weird to me to be using `o_request` in some of the parameters traced and `serverObject` in this new one. Made me check if we were tracing things from different places.

Reply:
Good point, done: https://github.com/Automattic/jetmon/pull/17/commits/8a15ca2f54f7480b9d12838f7677b7616bac80d2